### ISSUE TITLE
rip

### ISSUE THREAD

**Issue by TadusPro** [thomas-crane/nrelay#43]
https://i.imgur.com/SX52j4Z.png

**Comment by crinny** [comment]
> https://i.imgur.com/SX52j4Z.png

Having the same issue while connecting from Python so I assume this isn't the nrelay issue

**Comment by crinny** [comment]
I don't think this is an issue with packet IDs as `[0] Bad message received` occurs right after HELLO packet and its id didn't change:
![image](https://user-images.githubusercontent.com/42045258/80315969-c2a39000-8803-11ea-811c-20166071b3bc.png)
